js/foundation.util.mediaQuery.js

```diff
@@ -105,6 +113,16 @@ var MediaQuery = {
     this._watcher();
   },
 
+  /**
+   * Reinitializes the media query helper what might be necessary if your CSS is not (completely) loaded while _init() is executed.
```

Comment:
```
Reinitializes the media query helper.
Useful if your CSS breakpoint configuration has just been loaded or has changed since the initialization.
```